security/pkg/pki/util/keycertbundle.go

```diff
@@ -253,19 +253,18 @@ func (b *KeyCertBundle) UpdateVerifiedKeyCertBundleFromFile(certFile string, pri
 	return nil
 }
 
-// ExtractRootCertExpiryTimestamp returns the unix timestamp when the root becomes expires.
-func (b *KeyCertBundle) ExtractRootCertExpiryTimestamp() (float64, error) {
+// ExtractRootCertExpiryTimestamp returns when the root cert expires
```

Comment:
I will be explicit about what type of timestamp is being returned. This will help the istio.io user immediately know what to expect without looking at the code. 

`// ExtractRootCertExpiryTimestamp returns when the root cert expires as a *time.Time object.`